apps/meteor/client/startup/accounts.ts

```diff
@@ -24,3 +25,15 @@ Accounts.onEmailVerificationLink((token: string) => {
 		});
 	});
 });
+
+Meteor.startup(() => {
+	Tracker.autorun(() => {
+		const forgetUserSessionOnWindowClose = settings.get('Accounts_ForgetUserSessionOnWindowClose');
+
+		if (forgetUserSessionOnWindowClose === undefined) {
```

Comment:
Checking for undefined, as during the first run, the setting is `undefined`.

We don't want to unnecessarily configure the `Accounts` package, so returning here if it returns `undefined`.